### PR TITLE
Adding .npmignore file

### PR DESCRIPTION
**Pull request by ianwalter**
@substack Apparently this module is quite popular with other NPM authors. I have 202 copies of "In Praise of Idleness" by Bertrand Russell on my computer 😃

### CODE REVIEW

**Comment by jorrit** [comment]
You can also add .npmignore to the file itself.

**Comment by phpistasty** [comment]
This would be great. @substack - Any chance to get this change merged?

**Comment by Jaco-Pretorius** [comment]
+1

**Comment by juanformoso** [comment]
+1 will free up a lot of collective disk space across the world

**Comment by z-lake** [comment]
keep it 
everyone could use a reminder to enjoy their leisure time

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+example/
```
Comment (calvinmetcalf) [inline .npmignore]: this one is pretty small and might be useful for some people, if you really feel strongly about this folder being removed too maybe due a separate pull for removing this so any debate about it doesn't hold up the more straight forward test/ folder

**Comment by pakastin** [comment]
+1

**Comment by samueljseay** [comment]
So I created a PR before checking all the comments here, whoops. Anyway I see that some were keen to have the example folder retained so, I've created a new PR here that just excludes test: https://github.com/substack/node-wordwrap/pull/18 If its not at all helpful let me know.

File: .npmignore (hunk 1 above)
Comment (ianwalter) [inline .npmignore]: Looks like other people did this already, but I like this change better since the same exact examples are in the README. :beers:

**Comment by chitty** [comment]
Is this going to be merged @substack?

**Comment by substack** [comment]
Pretty small though:

```
$ curl -sS https://registry.npmjs.org/wordwrap/-/wordwrap-1.0.0.tgz | wc -c
14509
```

Generally speaking, I prefer to include tests and examples in packages. Github won't be around forever and it's handy to have tests and examples while working offline.

**Comment by Daniel15** [comment]
> Github won't be around forever

I mean, that's one of the reasons the code is in a Git repo, right? The entire purpose of Git is that it's decentralised.

Also, I feel like Github will outlive npm :)

**Pull request closed without merging**